<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CarControl</class>
 <widget class="QDialog" name="CarControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>361</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>小车连接</string>
   </property>
   <widget class="QRadioButton" name="radioButton_Car1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>71</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>小车1</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Car2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>71</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>小车2</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>361</width>
      <height>30</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>356</width>
      <height>89</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,7">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>IP地址：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_carIP"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="3,7">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>端口：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_carPort"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_startConnect">
         <property name="text">
          <string>确认连接</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="2,4,2,4">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>本机IP：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_selfIP">
         <property name="text">
          <string>NaN</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>本机端口：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_selfPort"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>171</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>方向盘</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>37</x>
      <y>20</y>
      <width>101</width>
      <height>100</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_Up">
       <property name="text">
        <string>↑</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton_Left">
       <property name="text">
        <string>←</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButton_Stop">
       <property name="text">
        <string>■</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pushButton_Right">
       <property name="text">
        <string>→</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButton_Down">
       <property name="text">
        <string>↓</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>180</y>
     <width>171</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>当前捕获红绿灯状态</string>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>红绿灯状态</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_light">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string> /</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>361</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>功能区</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>257</width>
      <height>136</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="pushButton_carStart">
         <property name="text">
          <string>启动</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_carStop">
         <property name="text">
          <string>停止</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,1">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="pushButton_carDistanceOn">
           <property name="text">
            <string>开启保持车距</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_carDistanceOff">
           <property name="text">
            <string>关闭保持车距</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>车载影像</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,1">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>防撞车距：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_CarDistance"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QTextEdit" name="textEdit_info">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>516</y>
     <width>361</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>500</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>信息：</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
